src/wow64ext.h

```diff
@@ -351,6 +351,10 @@ struct _CONTEXT64
 
 #pragma pack(pop)
 
+// Without the double casting, the pointer is sign extended, not zero extended,
+// which leads to invalid addresses with /LARGEADDRESSAWARE.
+#define PTR_TO_DWORD64(p) ((DWORD64)(DWORD)(p))

```

Comment:
If this is ever called from x64 code, it will truncate the pointer

Reply:
Yes, wow64ext is an x32-only library, but `PTR_TO_DWORD64` might be helpful in generic code outside of the library. I changed the macro, should be fine now, thanks.

Reply:
for me its ok, @rwfpl can you merge this PR?